orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -420,6 +420,7 @@ class OperationsController {
         noUserConfigurableFields: true,
         parameters: it.parameters,
         producesArtifacts: it.producesArtifacts,
+        ui: it.ui
```

Comment:
This needs to be updated for `uiType` change.

Reply:
groovy, thanks